src/levels/level-A.js

```diff
@@ -82,11 +82,16 @@ const LevelA = ({ squares, size, isStarted, isRestart, setRestartComplete }) =>
   const [begin, setBegin] = useState(false)
 
   useEffect(() => {
-    if (checkIfComplete(currentSquares)) {
-      setIsComplete(true)
+    const complete = checkIfComplete(currentSquares);
```

Comment:
this code is doing the same thing, I would leave it as it was so we're not setting an unnecessary variable. The function will automatically return that same bool you're setting as "complete". The bug is in the if statement, we need to add two more checks; one is that isStarted needs to be true and isRestart needs to be false (this is set back to false after the fade in / fade out)

Reply:
This works with restart and the fade!